raiko-guest/src/one_shot.rs

```diff
@@ -0,0 +1,227 @@
+use std::{
+    fs::{self, File, OpenOptions},
+    io::prelude::*,
+    path::PathBuf,
+    str::FromStr,
+};
+
+use anyhow::{anyhow, bail, Error, Result};
+use secp256k1::{hashes::sha256, rand::rngs::OsRng, All, Message, PublicKey, Secp256k1, SecretKey};
+use zeth_lib::{
+    block_builder::{TaikoBlockBuilder, TaikoStrategyBundle},
+    consts::TAIKO_MAINNET_CHAIN_SPEC,
+    host::Init,
+};
+
+use crate::app_args::{GlobalOpts, OneShotArgs};
+
+pub const ATTESTATION_QUOTE_DEVICE_FILE: &str = "/dev/attestation/quote";
+pub const ATTESTATION_TYPE_DEVICE_FILE: &str = "/dev/attestation/attestation_type";
+pub const ATTESTATION_USER_REPORT_DATA_DEVICE_FILE: &str = "/dev/attestation/user_report_data";
```

Comment:
These device file exist only on SGX-enabled machine running Gramine (another reason why keeping `no_sgx` mode is troublesome).